gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -369,6 +387,21 @@ public synchronized void setActive(boolean active) {
         for (Dag<JobExecutionPlan> dag : dags) {
           addDag(dag, false, false);
         }
+        if (dagActionStore.isPresent()) {
+          Collection<DagActionStore.DagAction> dagActions = dagActionStore.get().getDagActions();
```

Comment:
Do we need to worry about the scenario if we have multiple dagManagers or is that not in scope of this PR?

Reply:
No, That will be future work, this PR is dedicated to make sure we can do warm standby and can forward kill/resume message